v2/routes/instructorAPI.js

```diff
@@ -1037,6 +1037,38 @@ function instructorAPI(db, io, instructorAuth, isInstructor) {
 		}
 	});
 
+	/**
+	 * get the page stats (view count and time viewed) for a slide
+	 * req query:
+	 *   slideID: object ID of a slide
+	 */
+	router.get('/api/pageStats', instructorAuth, async (req, res) => {
```

Comment:
I think it is probably more useful to send stats of the whole file, otherwise, we will need a lot of back and forth for displaying a chart

Reply:
Poor api name lol. it sends info for the entire file. The return is an array where each index is {viewCount:#, timeViewed:#} and each index corresponds to a page. 

I'll rename this to slideStats

Reply:
ohhh lol I definitely didn't read it carefully